internal/cmd/db_list.go

```diff
@@ -31,29 +35,64 @@ var listCmd = &cobra.Command{
 			return err
 		}
 
-		data := [][]string{}
+		ctx, cancel := context.WithTimeout(context.Background(), 30*time.Second)
+		defer cancel()
+		g, ctx := errgroup.WithContext(ctx)
+		dbInfo := make(chan []string, len(databases))
+
 		for _, database := range databases {
-			url := getDatabaseUrl(settings, &database, false)
-			regions := getDatabaseRegions(database)
-
-			instances, err := client.Instances.List(database.Name)
-			if err != nil {
-				return err
-			}
-
-			token, err := client.Databases.Token(database.Name, "1d", true)
-			if err != nil {
-				return err
-			}
-			var size string
-			sizeInfo, err := calculateInstancesUsedSize(instances, settings, database, token)
-			if err != nil {
-				size = fmt.Sprintf("fetching size failed: %s", err)
-			} else {
-				size = sizeInfo.PrintTotal()
-			}
-			data = append(data, []string{database.Name, regions, url, size})
+			database := database
+			g.Go(func() error {
+				url := getDatabaseUrl(settings, &database, false)
+				regions := getDatabaseRegions(database)
+				g, _ := errgroup.WithContext(ctx)
+				instancesCh := make(chan []turso.Instance, 1)
+				g.Go(func() error {
+					instances, err := client.Instances.List(database.Name)
+					if err != nil {
+						return err
+					}
+					instancesCh <- instances
+					return nil
+				})
+
+				tokenCh := make(chan string, 1)
+				g.Go(func() error {
+					token, err := client.Databases.Token(database.Name, "1d", true)
+					if err != nil {
+						return err
+					}
+					tokenCh <- token
+					return nil
+				})
+				if err := g.Wait(); err != nil {
+					return err
+				}
+				instances := <-instancesCh
+				token := <-tokenCh
+				var size string
+				sizeInfo, err := calculateInstancesUsedSize(instances, settings, database, token)
+				if err != nil {
+					size = fmt.Sprintf("fetching size failed: %s", err)
+				} else {
+					size = sizeInfo.PrintTotal()
+				}
+				dbInfo <- []string{database.Name, regions, url, size}
+				return nil
+			})
+		}
+
+		if err := g.Wait(); err != nil {
+			return err
+		}
+
+		var data [][]string
+		for range databases {
+			data = append(data, <-dbInfo)
 		}
+		sort.Slice(data, func(i, j int) bool {
+			return data[i][0] > data[j][0]
+		})
```

Comment:
:pick:: You can use the index of the for loop to populate the data table directly, removing the need for the `dbInfo` channel.